CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations
 on how to structure this file.
 This plugin uses [Semantic versioning](https://semver.org).
 
+## [2.9.0] - 2021-11-23
```

Comment:
To make this work, we need to make sure the commit message when merging includes either "feat" or "minor". See also https://github.com/phips28/gh-action-bump-version which is used in the CD workflow

Reply:
It should be sufficient to have one commit in this PR that includes either of those keywords.

Reply:
I added a commit with the name `feature: implemented Go to Definition` :+1:

Reply:
Should work :)